assets/js/src/common/button/button.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classnames from 'classnames';
+
+type Props = {
+  children: React.ReactNode,
+  size?: 'small' | 'large',
+  variant?: 'light' | 'dark' | 'link' | 'link-dark',
+  isLoading?: boolean,
```

Comment:
I think this is not a great name. The button is not loading anything. The button has a spinner, so maybe a better name would be `withSpinner`

The same applies to the class names.